src/GuestPages/RSVP.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Status from './Status';
+import NewRsvp from './NewRSVP';
+
+const RsvpForm = () => (
+  <Container fluid>
+    <Container fluid>
```

Comment:
Does this element need a second container? Just something to consider, but disregard if this breaks your ability to style.

Reply:
This will be addressed in my refactor